### DISCUSSION TITLE
Set `envlogdir` from environment variable

### DISCUSSION THREAD

**Discussion by abitrolly** [tox-dev/tox#2094] [Q&A] [answered]
I need to configure location of `envlogdir` using environment variables, and can not find any example. Is it supported?

**Comment by gaborbernat** [comment, marked as answer]
I don't think this is possbile today.

**Reply by siddharthmohan0998-crypto** [reply]
Start new code for international marketing and affiliate marketing  this is a best for results and targeting a new step count
